src/app/components/evento-tarjeta/evento-tarjeta.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import { Router } from '@angular/router';
+import { Evento } from 'src/app/models/evento';
+
+
+@Component({
+  selector: 'app-evento-tarjeta',
+  templateUrl: './evento-tarjeta.component.html',
+  styleUrls: ['./evento-tarjeta.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138940)
```suggestion
  styleUrls: ["./evento-tarjeta.component.css"]
```